test/e2e/azure_test.go

```diff
@@ -468,6 +468,65 @@ var _ = Describe("Workload cluster creation", func() {
 		})
 	})
 
+	// ci-e2e.sh and Prow CI skip this test by default.
+	// To include this test, set `GINKGO_SKIP=""`.
+	Context("Creating a cluster with user-assigned identity [OPTIONAL]", func() {
+		It("with a 1 control plane nodes and 0 worker nodes", func() {
```

Comment:
@mboersma the LB spec won't work with 0 worker nodes, we need worker nodes to create a valid deployment with running pods